migrations/20161116101047-create-comment.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('Comments', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      content: {
+        type: Sequelize.TEXT
```

Comment:
I could add allowNull: false here to also put a NULL Constraint on the content this way database would reject any comment insertion where the content is empty.